Microsoft.RestApi.RestTransformer/RestOperationTransformer.cs

```diff
@@ -348,6 +351,46 @@ private static IList<PathEntity> TransformPaths(RestApiChildItemViewModel viewMo
             return pathEntities;
         }
 
+        private static string TransformPermissions(RestApiChildItemViewModel viewModel, IList<PathEntity> pathEntities)
+        {
+                var pathEntity = pathEntities.FirstOrDefault();
+                var index = pathEntity.Content.LastIndexOf("/providers/");

```

Comment:
What if index == -1

Reply:
Don't do anything about it

for example
![image](https://user-images.githubusercontent.com/66710086/111125674-d89d7900-85ac-11eb-9271-405a25ad7524.png)

I want to get this string("/Microsoft.PolicyInsights/policyEvents/default/queryResults") and use it for regular verification